### ISSUE TITLE
Unable to build git

### ISSUE THREAD

**Issue by FallingSnow** [servo/devices#22]
```
$ rustc -vV
rustc 1.25.0-nightly (3d292b793 2018-02-03)
binary: rustc
commit-hash: 3d292b793ade0c1c9098fb32586033d79f6e9969
commit-date: 2018-02-03
host: x86_64-unknown-linux-gnu
release: 1.25.0-nightly
LLVM version: 4.0
```

```rust
   Compiling device v0.0.1 (https://github.com/servo/devices#c3b012b0)
error[E0061]: this function takes 1 parameter but 0 parameters were supplied
   --> /home/ayrton/.cargo/git/checkouts/devices-1d074a12853a87bb/c3b012b/src/bluetooth.rs:951:64
    |
951 |         get_inner_and_call!(self, BluetoothGATTCharacteristic, read_value)
    |                                                                ^^^^^^^^^^ expected 1 parameter

error[E0061]: this function takes 2 parameters but 1 parameter was supplied
   --> /home/ayrton/.cargo/git/checkouts/devices-1d074a12853a87bb/c3b012b/src/bluetooth.rs:955:64
    |
955 |         get_inner_and_call!(self, BluetoothGATTCharacteristic, write_value, values)
    |                                                                ^^^^^^^^^^^ expected 2 parameters

error[E0061]: this function takes 1 parameter but 0 parameters were supplied
    --> /home/ayrton/.cargo/git/checkouts/devices-1d074a12853a87bb/c3b012b/src/bluetooth.rs:1051:60
     |
1051 |         get_inner_and_call!(self, BluetoothGATTDescriptor, read_value)
     |                                                            ^^^^^^^^^^ expected 1 parameter

error[E0061]: this function takes 2 parameters but 1 parameter was supplied
    --> /home/ayrton/.cargo/git/checkouts/devices-1d074a12853a87bb/c3b012b/src/bluetooth.rs:1055:60
     |
1055 |         get_inner_and_call!(self, BluetoothGATTDescriptor, write_value, values)
     |

error: aborting due to 4 previous errors

error: Could not compile `device`.
```

**Comment by jdm** [comment]
Yes. It appears that when blurz 0.2.3 was released, it updated some APIs that this crate relied upon. It should have been published as 0.3.0 instead to avoid this problem. You can work around it by downgrading blurz to 0.2.2.

**Comment by SimonSapin** [comment]
This is fixed: https://github.com/szeged/blurz/pull/17
